### PR TITLE
chore: add AWS link and GCP to the Kafka overview page

### PR DESCRIPTION
**Pull request by Kells512**
**What this PR does / why we need it**:

>Making updated to the Kafka Overview page, and making version 3. 
>Changes found in document: https://docs.google.com/document/d/1zhkNbi5pI2_tc2USqbkcm0MFnuOKg97TvtEWwiEUErk/edit

**Verification steps** 

> APPSERVICES > <Kafka Page V 3> 

> Updating Purchase now and Contact sales cards
>**MGDSTRM-9141: add link to AWS Marketplace to Kafka Overview page:** https://issues.redhat.com/browse/MGDSTRM-9141

Screenshot from 2022-07-14 16-44-45

>Adding GCP to Cloud Providers and make layout more responsive at smaller viewpoints
>**MGDSTRM-9119: Add GCP to the Kafka overview page:** https://issues.redhat.com/browse/MGDSTRM-9119

> **With Vertical dividers and rows**
Screenshot from 2022-07-15 09-14-02

> **With Horizontal dividers and columns**
Screenshot from 2022-07-18 09-19-04

### CODE REVIEW

**Comment by Kells512** [comment]
@jgiardino the changes are here, had to open new PR

File: locales/en/kafkaoverview-v3.json
Comment (jgiardino) [inline locales/en/kafkaoverview-v3.json:24]: There's still a typo in "Canada" here

Comment (Kells512) [reply locales/en/kafkaoverview-v3.json]: oh sorry that's fixed now

**Review by jgiardino — APPROVED** [review]

File: src/AppServices/KafkaPageV3.tsx
Comment (jgiardino) [reply src/AppServices/KafkaPageV3.tsx:288]: I didn't mean to block this PR from merging with this comment. I updated the review status to approved. If this comment is an easy fix, let's add it. But if it's not obvious, we can merge without it.

Comment (Kells512) [reply src/AppServices/KafkaPageV3.tsx:288]: I think it could be an easy fix, I'll try it now

Comment (Kells512) [reply src/AppServices/KafkaPageV3.tsx:288]: I can't seem to get it to look exactly like it did before, so maybe it's better to merge without it for now

Comment (jgiardino) [reply src/AppServices/KafkaPageV3.tsx:288]: No worries, let's merge as-is.

**Review by anukritijha — APPROVED** [review]
LGTM

**Comment by anukritijha** [comment]
@riccardo-forina @ajaypratap003 Is there any reason for versioning the files (KafkaPage, KafkaPageV2, KafkaPageV3) in AppServices?

**Comment by HemaHG** [comment]
There were some changes to kafkaPage during the summit, so we created kafkaPageV2 so that if something goes wrong with the latest changes it will be easy to revert it back

**Comment by anukritijha** [comment]
> There were some changes to kafkaPage during the summit, so we created kafkaPageV2 so that if something goes wrong with the latest changes it will be easy to revert it back

Does it make sense to just have V3 as KafkaPage now. And similarly for all the other files in AppServices?

**Comment by riccardo-forina** [comment]
Probably not. Let's keep it since it's done, and follow up after we go on production with a PR that removes the unused versions.

**Comment by anukritijha** [comment]
> Probably not. Let's keep it since it's done, and follow up after we go on production with a PR that removes the unused versions.

sounds good.

**Pull request merged**
